sched/generators.go

```diff
@@ -0,0 +1,51 @@
+package sched
+
+import (
+	"fmt"
+	"github.com/leanovate/gopter"
+	//"github.com/leanovate/gopter/gen"
+)
+
+func GenJob(id string, numTasks int) Job {
+	jobDef := JobDefinition{
+		JobType: "testJob",
+		Tasks:   make(map[string]TaskDefinition),
+	}
+
+	// Generate tasks
+	for i := 0; i < numTasks; i++ {
+		task := GenTask()
+		taskId := fmt.Sprintf("Task%v", i)
+		jobDef.Tasks[taskId] = task
+	}
+
+	job := Job{
+		Id:  id,
+		Def: jobDef,
+	}
+
+	return job
+}
+
+func GenTask() TaskDefinition {
```

Comment:
I think as the values applied to tasks gets used more, i.e. worker code gets implemented, then we can definitely make this more random as a generator.  Or we start doing datalocality based scheduling.